### ISSUE TITLE
Audio instructions vs written instructions

### ISSUE THREAD

**Issue by mcwhittemore** [Project-OSRM/osrm-text-instructions#175]
The more that I dive into audio instructions the more that I realize that good audio instructions and good written instructions are different things. This library seems capable of solving both if it exposed a function to compile written instructions and function to compile audio instructions rather than one `compile` function.

**Comment by bsudekum** [comment]
Agreed. Although grammatically correct, commas add some unnecessary pauses when an instruction is spoken.

**Comment by yuryleb** [comment]
Please believe me, combining prepared audio fragments with TTS-spoken way names sounds terrible.

Almost the same is already implemented in latest Gamin TTS voices - static instruction parts are "prepared" with HD-quality voice, way names are pronounced at runtime by low-quality one. You could suppose how it sounds -  different speed/tone for the same phrase words :disappointed_relieved:

**Comment by JasonDalton** [comment]
Is there any interest in getting this in the pipeline.  Providing spoken language text and banner text in addition to the written text.  Mirroring the MapBox output from map matching for instance would free up the use of OSRM routing to be used directly in MapBox Navigation SDK.
